src/arbitraries/UniqueArbitrary.ts

```diff
@@ -1,48 +0,0 @@
-import {FluentPick} from './types'
-import {Arbitrary, WrappedArbitrary} from './internal'
-import {stringify} from './util'
-
-export class UniqueArbitrary<A> extends WrappedArbitrary<A> {
-  constructor(readonly baseArbitrary: NonNullable<Arbitrary<A>>) {
-    super(baseArbitrary)
-  }
-
-  sample(sampleSize = 10, cornerCases: FluentPick<A>[] = []): FluentPick<A>[] {
-    // TODO: Here lies dragons! If you see start seeing things in double when
```

Comment:
@arestivo I noticed now you removed this comment. I think it's important to keep this caveat visible as unique only works if values are deemed equal under our own `stringify`, which uses custom logic.